src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -297,14 +330,36 @@ class TimelineMarkerCanvas extends PureComponent {
     y: CssPixels,
     width: CssPixels,
     height: CssPixels,
-    cornerSize: CssPixels
+    radius: CssPixels
   ) {
-    // Cut out c x c -sized squares in the corners.
-    const c = Math.min(width / 2, Math.min(height / 2, cornerSize));
-    const bottom = y + height;
-    ctx.fillRect(x + c, y, width - 2 * c, c);
-    ctx.fillRect(x, y + c, width, height - 2 * c);
-    ctx.fillRect(x + c, bottom - c, width - 2 * c, c);
+    // stolen from https://www.rgraph.net/canvas/reference/arcto.html
+
+    const r = Math.min(radius, height / 2, width / 2);
+    const x2 = width - 1;
+    const y2 = height - 1;
```

Comment:
This is one of the issues that we had before: `bottom` should have been `y + height - 1`, because yStart and yEnd are both part of the rectangle (inclusively).